File: Python/1036.py
import math

a, b, c = list(map(float,input().split()))

if a == 0 or math.pow(b, 2) < 4*a*c:
    print('Impossivel calcular')
else:

    r1 = (-b + math.sqrt(math.pow(b, 2) - 4*a*c))/(2*a)
    r2 = (-b - math.sqrt(math.pow(b, 2) - 4*a*c))/(2*a)

    print('R1 = {:.5f}'.format(r1))
    print('R2 = {:.5f}'.format(r2))

File: C/1036.c
#include <stdio.h>
#include <math.h>

int main() { 
    double a, b, c, r1, r2;

    scanf("%lf %lf %lf", &a, &b, &c);

    if(a == 0 || pow(b, 2) < 4*a*c){
        printf("Impossivel calcular\n");
    }else{
        r1 = (-b + sqrt(pow(b, 2) - 4*a*c))/(2*a);
        r2 = (-b - sqrt(pow(b, 2) - 4*a*c))/(2*a);

        printf("R1 = %.5lf\n", r1);
        printf("R2 = %.5lf\n", r2);
    }

    return 0;
}